Casks/hex-fiend.rb

```diff
@@ -4,11 +4,15 @@
 
   url "https://github.com/ridiculousfish/HexFiend/releases/download/v#{version}/Hex_Fiend_#{version.major_minor}.dmg",
       verified: "github.com/ridiculousfish/HexFiend/"
-  appcast "https://github.com/ridiculousfish/HexFiend/releases.atom"
   name "Hex Fiend"
   desc "Hex editor focussing on speed"
   homepage "https://ridiculousfish.com/hexfiend/"
 
+  livecheck do
+    url "https://github.com/ridiculousfish/HexFiend/releases.atom"
```

Comment:
```suggestion
    url :cask_url
```

We try to align the `livecheck` source with the `stable` URL in formulae, so we should aim to check as close to the `cask_url` as possible, too. There isn't any benefit to using the `.atom` URL here, since the `GithubLatest` strategy simply generates the `/releases/latest` URL from it (i.e., not checking the supplied URL directly).

Reply:
I'd like to change this to `:url`. Naming it `:cask_url` is redundant, given this is in a cask.

Reply:
Makes sense to me, since it's just referred to as `url` in cask objects (e.g., some_cask.url`). We use the `:stable` symbol to refer to `url` in formulae since you access that URL on a formula object like `some_formula.stable.url`.